server/monitor.go

```diff
@@ -976,6 +976,7 @@ type Varz struct {
 	HTTPHost          string            `json:"http_host"`
 	HTTPPort          int               `json:"http_port"`
 	HTTPSPort         int               `json:"https_port"`
+	BasePath          string            `json:"base_path"`
```

Comment:
Do we really need that? By definition, if you get to the /varz endpoint, it means that you provided the right path, no?

Reply:
Honestly, I'm not sure as well. Perhaps we don't need it, but then why we need `HTTPPort` and `HTTPSPort`? in the end, you also know the port we called, right? So far I believe they solve the same use-case, which I'm not sure what is, just wanted to keep consistent.

Reply:
You are right, I always had the same thinking when server sends host/port in INFO to clients :-). So leave it there for consistency.